RELEASING.md

```diff
@@ -32,3 +32,11 @@ Repositories create consistent release labels, such as `v1.0.0`, `v1.1.0` and `v
 ## Releasing
 
 The release process is standard across repositories in this org and is run by a release manager volunteering from amongst [MAINTAINERS](MAINTAINERS.md).
+
+1. Create a tag, e.g. `v2.1.0`, and push it to the GitHub repo.
+2. The [opensearch-java-maven-sign-and-release/](https://build.ci.opensearch.org/job/opensearch-java-maven-sign-and-release/) will be automatically kicked off.
+3. Login to [AWS OSS Sonatype](https://aws.oss.sonatype.org/#stagingRepositories), and locate the staged release.
+4. `Close` the staged repository, and ensure all checks pass.
+5. `Release` the staged respository.
```

Comment:
Do these steps really help the community ? 
If its internal, I would not have them listed down.

Reply:
Anyone with access to sonatype could do this, and it should really be the maintainers of the project. I think the right answer is to also automate this part as well if we're going to continue using this shared account.

Reply:
See https://github.com/opensearch-project/opensearch-build/issues/2398 for details. Maybe add it to readme till we resolve it?

Reply:
Makes sense.